imports/api/users.js

```diff
@@ -192,13 +207,27 @@ Meteor.methods({
    * update profile image with new image in S3 collection
    * @param {String} profileImageUrl
    */
-  'users.updateProfileImage' (profileImageUrl) {
+  'users.updateProfileImage' (profileImageUrl, storageType) {
```

Comment:
make this take the url and set it directly, as it was before. Only deal with the update in the getURL